toolchain/check/cpp/import.cpp

```diff
@@ -1157,7 +1157,7 @@ static auto LookupCustomRecordType(Context& context,
 // Maps a C++ tag type (class, struct, union, enum) to a Carbon type.
 static auto MapTagType(Context& context, const clang::TagType& type)
     -> TypeExpr {
-  auto* tag_decl = type.getOriginalDecl();
+  auto* tag_decl = type.getDecl();
```

Comment:
Looks like the version of LLVM we're moving to doesn't have this rename yet (https://github.com/llvm/llvm-project/commit/b516dcc998d06c97d874af543489887f7e5a680c landed on October 15, but this PR is picking [an LLVM revision from September](https://github.com/llvm/llvm-project/commit/113f01aa82d055410f22a9d03b3468fa68600589)).